### PR TITLE
add subscription_interval as config for dbal subscription consumer

### PR DESCRIPTION
**Pull request by mordilion**
I am facing an issue with the situation, that I have a few JobWorker running and my DB doesn't like that many requests. Without the change, I am getting 50 Requests in 10 Seconds with 1 JobWorker. Because of this, I am getting many "MySQL Server has gone away" messages.

### CODE REVIEW

**Comment by mordilion** [comment]
@makasim is it possible to get at least a short feedback for my Pull-Request? Right now I am patching my Code every time to get that option. At least a "No" would be nice, because then I could start looking for another Library.

In my opinion, it would make sense to add that option. :)

File: pkg/dbal/DbalSubscriptionConsumer.php [hunk 1]
```diff
@@ -36,6 +36,13 @@ class DbalSubscriptionConsumer implements SubscriptionConsumer
      */
     private $redeliveryDelay;
 
+    /**
+     * Time to wait between subscription requests in milliseconds.
+     *
+     * @var int
+     */
+    private $subscriptionInterval = 200;
```
Comment (makasim) [inline pkg/dbal/DbalSubscriptionConsumer.php:44]: What about naming it pollingInterval like [here](https://github.com/php-enqueue/enqueue-dev/blob/master/pkg/fs/FsConsumer.php#L39) to stay consistent with other transports ? 

The tests should be fixed

**Review by makasim — CHANGES_REQUESTED** [review]

**Comment by makasim** [comment]
In general it looks good.

**Review by mordilion — COMMENTED** [review]
Changed the naming :)

**Pull request merged**